docs/fr/guide/essentials/nested-routes.md

```diff
@@ -87,7 +87,7 @@ const router = new VueRouter({
       children: [
         // `UserProfile` va être rendu à l'intérieur du `<router-view>` de `User`
         // quand `/utilisateur/:id` concorde
-        { path: '', component: UserHome },
+        { path: '', component: UserProfile },
```

Comment:
It's the other way around, `UserProfile` should be `UserHome`

Reply:
I did this way to match the english documentation